docs/manuals/core/config/core-config.md

```diff
@@ -21,22 +15,31 @@ Here are the (optional) configurable properties:
   If _**repository_type**_ is set to `filesystem`, Taipy uses _**root_folder**_ and _**storage_folder**_ to store
   entities. In this case, _**repository_properties**_ attribute is not required.</br>
   If _**repository_type**_ is set to `sql`, the _**db_location**_ attribute is required as the path of a sqlite3
-  database file. Please refer to [SQL storage section](global-config.md#sql-storage-for-taipy-entities).</br>
+  database file. Please refer to [SQL storage section](core-config.md#sql-storage-for-taipy-entities).</br>
   If _**repository_type**_ is set to `mongo`, the possible properties are _**mongodb_hostname**_, _**mongodb_user**_,
   _**mongodb_password**_, _**mongodb_port**_, _**application_db**_, _**properties**_. Please refer to
-  [MongoDB storage section](global-config.md#mongodb-storage-for-taipy-entities).
+  [MongoDB storage section](core-config.md#mongodb-storage-for-taipy-entities).
+- _**mode**_: A string that indicates the mode of the version management system.
+  Possible values are *"development"*, *"experiment"*, or *"production"*.
+- _**version_number**_: The identifier of the version. In development mode, the version number is ignored.
+- _**force**_: Indicates whether Taipy will override a version even if the configuration has changed
+    or not and run the application. Default to False.
+- _**clean_entities**_: Indicates whether running a Taipy Core service will clean all current version
+    entities or not before running the application. Default to False.
```

Comment:
Sorry, I missed that in the other PRs for the ticket [core-#666](https://github.com/Avaiga/taipy-core/issues/666). Probably because it s a Demoniac ticket. 🤣

In the ticket, the field `clean_entities` should be completely removed from the config.
Since we have the development mode that does it automatically, and since we have the CLI to manage versions, it is not needed anymore. Plus, it is error-prone and can be dangerous to remove all production entities just with a config value.
I discussed with @FlorianJacta that agrees with that.

That means we need to remove the attribute from the `CoreSection`, and it also means we need to remove lines 205, 206, and 207 in `taipy-core/src/taipy/core/taipy.py`

```suggestion
```

Reply:
We need to update some example, and the code on taipy-core for this also. Do you want to do it in another ticket?

Reply:
I don't mind. It s up to you.

Reply:
Ticket created: https://github.com/Avaiga/taipy-core/issues/693